Detectors/TRD/base/include/TRDBase/TrackletTransformer.h

```diff
@@ -61,15 +57,15 @@ class TrackletTransformer
   double getTimebin(int detector, double x);
 
  private:
-  o2::trd::Geometry* mGeo;
-  const o2::trd::PadPlane* mPadPlane;
+  Geometry* mGeo{nullptr};
+  const PadPlane* mPadPlane{nullptr};
 
   float mXCathode;
   float mXAnode;
   float mXDrift;
   float mXtb0;
 
-  o2::trd::CalVdriftExB* mCalibration;
+  const CalVdriftExB* mCalibration{nullptr};
```

Comment:
We will need some other calibrations in this class as well, mainly for PID: the krypton gain tables and the chamber gain. 
I would suggest to rename this member variable in preparation for that.
```suggestion
  const CalVdriftExB* mCalVdriftExB{nullptr};
```

Reply:
Why would we actually need those PID calibrations in this class? The CalibratedTracklet only stores position information and one direction. I thought this would be covered by vDrift and ExB? But T0 might come at some point, so I will do the naming change nevertheless